components/rpill/browser/rpill.cc

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include <locale>
+#include <string>
+
+#include "brave/components/rpill/browser/rpill.h"
+
+#include "base/strings/string_util.h"
+#include "base/win/wmi.h"
+#include "content/public/browser/browser_thread.h"
+
+namespace rpill {
+
+bool exec() {
+  DCHECK_CURRENTLY_ON(content::BrowserThread::IO);
+  base::win::WmiComputerSystemInfo info =
```

Comment:
in this method you should use `#if defined(OS_WIN)` and also for the header for win  even though this file is currently only called on windows. The callers should still use the buildflag. Other OS would just return true even though the code won't currently get called for them. It isn't strictly necessary in this case, but it's cleaner for the future